tensorflow/c/c_api_test.cc

```diff
@@ -1543,6 +1543,33 @@ TEST(CAPI, TestBitcastFrom_Reshape) {
   TF_DeleteTensor(b);
 }
 
+TEST(CAPI, TestFromProto) {
+  Tensor t_cc(DT_FLOAT, TensorShape({2, 3}));
+  t_cc.flat<float>().setConstant(1.0);
+  tensorflow::TensorProto t_proto;
```

Comment:
This needs an additional include:

`#include "third_party/tensorflow/core/framework/tensor.proto.h"`

Reply:
Done. And I guess what you actually mean is `#include "tensorflow/core/framework/tensor.pb.h"`